app/views/surveys/component/_form-footer.pug

```diff
@@ -3,10 +3,20 @@ section.form-footer
     if !headless
       .form-footer__content__main-controls  
         if type === 'preview'
+          if nextPrompt
+            fieldset.draft.question.next-prompt
```

Comment:
The "draft" class doesn't do anything I believe and is confusing. You can target this question with `.next-prompt` in CSS

Reply:
got it, style updated